main.py

```diff
@@ -192,7 +38,15 @@ def send_text_or_list(self, update: Update, context: CallbackContext, text: Unio
             return
         if isinstance(text, list):
             text = "\n".join(text)
-        context.bot.send_message(chat_id=update.effective_chat.id, text=text, parse_mode=parse_mode)
+        try:
+            context.bot.send_message(chat_id=update.effective_chat.id, text=text, parse_mode=parse_mode)
+        except Exception as e:
+            if parse_mode == ParseMode.MARKDOWN:
+                logging.error(f"Unable to send message; possibly Markdown issue due to caller not using markdown_escape(). Trying again with formatting disabled.")
```

Comment:
Why not `logging.exception()` ?  It allows custom message + adds information about the exception with the traceback.

Reply:
good one. fixed